blockchain/blockchain-wallet-v4-frontend
# feat/persistent-alerts

## Description
1205

## Change Type
Please enter one or more of the following: 
- Upgrades

## Testing Steps
1. create wallet
2. verify email
3. logout and login
4. see auth required message persists until authorized

## Code Checklist
